parser.go

```diff
@@ -213,6 +215,9 @@ func (p *parser) readValue(in []byte, ignoreContinuation, ignoreInlineComment bo
 		if pos == -1 {
 			return p.readMultilines(line, line[startIdx:], valQuote)
 		}
+		if valQuote == `"` {
+			return strings.Replace(line[startIdx:pos+startIdx], `\"`, `"`, -1), nil
```

Comment:
How do you distinguish user actually wants `\"` instead of replacing them to `"`"?